src/components/SpectralProfiler/SpectralProfilerComponent.tsx

```diff
@@ -49,6 +54,15 @@ export class SpectralProfilerComponent extends React.Component<WidgetProps> {
         console.log("can't find store for widget");
         return new SpectralProfileWidgetStore();
     }
+    // not sure if I need this
+    @computed get profileStore(): SpectralProfileStore {
+        const widgetStore = this.widgetStore;
+        if (widgetStore.effectiveFrame) {
+            //const profileKey = `${widgetStore.effectiveFrame.frameInfo.fileId}-${widgetStore.effectiveRegionId}`;
```

Comment:
please remove commented code

Reply:
I need to look at this more closely. This might be why @kswang1029 sees the crash when undocking. For now I have left it commented because the code crashes without the function, does nothing but works when the code is commented and fails when it is uncommnented. I didn't write this and don't understand how to fix the issue yet.

```
/export/home/fornax/jhoskins/Development/CARTA/carta-frontend/src/components/SpectralProfiler/SpectralProfilerComponent.tsx
TypeScript error in /export/home/fornax/jhoskins/Development/CARTA/carta-frontend/src/components/SpectralProfiler/SpectralProfilerComponent.tsx(63,13):
Type 'ObservableMap<number, SpectralProfileStore>' is missing the following properties from type 'SpectralProfileStore': fileId, regionId, profiles, getProfile, and 2 more.  TS2740

    61 |         if (widgetStore.effectiveFrame) {
    62 |         //    const profileKey = `${widgetStore.effectiveFrame.frameInfo.fileId}-${widgetStore.effectiveRegionId}`;
  > 63 |             return AppStore.Instance.spectralProfiles.get(widgetStore.effectiveFrame.frameInfo.fileId);
       |             ^
    64 |         }
    65 |         return undefined;
    66 |     }
```